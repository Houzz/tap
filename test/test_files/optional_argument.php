<?php

/**
 * @param int $a
 * @param int|null $b
 */
function checkOptionalParam($a, $b = null) {}

checkOptionalParam(3);
checkOptionalParam(3, 4);
checkOptionalParam(3, 4, 5);    // error, too more args
