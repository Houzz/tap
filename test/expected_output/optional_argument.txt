11 E_FUNCTION_ARGUMENT
